api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -33,11 +34,13 @@
 	private String name;
 	
 	private String description;
-	
+
 	private Set<Integer> memberIds;
-	
+
+	private Collection<CohortMembership> members;
+
 	public Cohort() {
-		memberIds = new TreeSet<Integer>();
+        members = new TreeSet<CohortMembership>();
```

Comment:
@dkayiwa I just checked my code, strangely it appears fine in my IDE, but not in a text editor:

![screen shot 2016-12-05 at 12 23 00 pm](https://cloud.githubusercontent.com/assets/193819/20901064/aa32add8-bae5-11e6-9ecf-ade3537c2ee6.png)

Has this happened to you?